src/Http/Http.Abstractions/src/StatusCodes.cs

```diff
@@ -283,6 +283,11 @@ public static class StatusCodes
     /// </summary>
     public const int Status451UnavailableForLegalReasons = 451;
 
+    /// <summary>
+    /// HTTP status code 499.
+    /// </summary>
+    public const int Status499ClientClosedRequest = 499;
```

Comment:
For completeness, please add this to ReasonPhrases as well:
https://github.com/dotnet/aspnetcore/blob/249dfcb2bc5dd30152ce10c47e8c77f2cc95ca81/src/Http/WebUtilities/src/ReasonPhrases.cs#L68-L69